.github/workflows/tf-plan-apply.yaml

```diff
@@ -0,0 +1,94 @@
+name: Terraform Plan & Apply On Successful PR
+
+on:
+  workflow_call:
+    secrets:
+      GITHUB_OWNER:
```

Comment:
can we add these at the project scope so this workflow is reusable for all terraform not just the github specific terraform

Reply:
Adding these to project workflow does not work as passing [environment variable to re-usable workflows does not work without having some kind of hack](https://github.com/orgs/community/discussions/26671). Other options would be to encrypt these values in the project. I changed the name and made it more clear and not required